master/static/migrations/20240507113938_add-workspace-namespace-bindings-table.tx.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE public.workspace_namespace_bindings (
+    workspace_id INT REFERENCES workspaces(id) ON DELETE CASCADE,
+    cluster_name text NOT NULL,
+    namespace_name text NOT NULL
```

Comment:
primary key over (workspace_id, cluster_name, namespace)? (or since that will make an index, order it so that it works with your queries so they get a free index only scan)

Reply:
Done